### PR TITLE
UNOMI-419: fix SegmentsIT test to wait on full startup

### PR DESCRIPTION
**Pull request by jkevan**

### CODE REVIEW

**Review by dgriffon — APPROVED** [review]

File: itests/src/test/java/org/apache/unomi/itests/SegmentIT.java [hunk 1]
```diff
@@ -46,8 +47,15 @@
     @Filter(timeout = 600000)
     protected SegmentService segmentService;
 
+    @Inject @Filter(timeout = 600000)
+    protected BundleWatcher bundleWatcher;
+
     @Before
     public void setUp() throws InterruptedException {
+        while (!bundleWatcher.isStartupComplete()) {
```
Comment (sergehuber) [inline itests/src/test/java/org/apache/unomi/itests/SegmentIT.java:55]: Ideally we should move this setup to BaseIT since all the tests should have it, but we should do this in a separate PR.
